app/views/devise/registrations/new.html.erb

```diff
@@ -1,6 +1,27 @@
-<h2>Sign up</h2>
 
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+<div class="flex min-h-full flex-col justify-center py-12 sm:px-6 lg:px-8">
+
+
+
+  <div class="sm:mx-auto sm:w-full sm:max-w-md">
+    <div class="mx-auto h-36 w-auto text-center text-8xl">
+      <img src="https://d2t1xqejof9utc.cloudfront.net/screenshots/pics/ccd583a310fc605f82c1f2345fadb0a7/large.png" class="object-contain h-36 w-full object-center" />
+    </div>
+      <h2 class="text-center text-5xl font-bold tracking-tight text-gray-900 uppercase mt-8">
+        Sign Up
```

Comment:
We should keep the original brand name here as in the login page
![image](https://user-images.githubusercontent.com/73831187/197022812-cec895ae-21db-413b-a8a7-3c9563f44ef3.png)

Reply:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/42187168/197037249-cc067a0b-8da1-45bb-bc33-53bebb035b21.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/42187168/197037266-edc05c61-24ac-42d4-a844-cfcbd45035fe.png">